nb/ide.branding/core.windows/src/org/netbeans/core/windows/options/Bundle_nb.properties

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+# Space separated list of LAF classes to filter from UI
+Laf_Filter=MetalLookAndFeel \
```

Comment:
Perhaps find a better name than "Filter"; it's not clear from the name if it means include or exclude. Perhaps "Hidden_LAFs"?

Reply:
Also, maybe reverse the sense of the filter; instead of listing the LAFs to include, list LAFs to exclude. If new LAFs appear for any reason, it's likely useful to show them.

Reply:
It already is an exclusion filter for that very reason (eg. user installs LaF plugin).  I'll rename to make that clearer, although as I'd like to keep the Laf prefix, I'll use `Laf_Excludes`.

Reply:
Oh, sorry, yeah you're right... I got the include/exclude sense mixed up in my head. Yeah, good call with the Laf_ prefix.